### ISSUE TITLE
iOS: cannot report posts or accounts

### ISSUE THREAD

**Issue by nstrong-scw** [bluesky-social/social-app#774]
**Describe the bug**
When attempting to report a post or account on iOS, the "Send Report" button is not accessible because it is off-screen and cannot be scrolled to.

**To Reproduce**

1. Launch the app
2. Tap on a post
3. Tap on the "..." and tap "Report Post"
4. Select a report category

**Expected behavior**

The "Send Report" button should appear so I can submit the report

**Actual Behavior**
The "Send Report" button is not visible at all and cannot be scrolled to. I can barely see it if I grab the handle on the report and drag it upwards (see screenshot).

**Screenshots**
JUST TO BE CLEAR, THIS IS A DEMO AND I AM NOT ACTUALLY TRYING TO REPORT THIS SPECIFIC POST!
![IMG_2635](https://github.com/bluesky-social/social-app/assets/102557081/25270d25-3952-406b-aec1-a3faf9ccfb65)
![IMG_2636](https://github.com/bluesky-social/social-app/assets/102557081/2d6c6a94-8916-4ba3-91b7-ba0d3cb4742a)

**Details**

 - Platform: iOS, iPadOS
 - Platform version: 16.4.1 (a)
 - App version: 1.28.0 (3)

**Additional context**
As a workaround, I can log into the web app via Safari, and the report process works as expected using that front-end.

**Comment by ansh** [comment]
Which iPhone do you have? Also, have you made any changes to your font size?

**Comment by nstrong-scw** [comment]
Thanks for replying!

- iPhone 12 mini
- iPad Air 4th gen

on my iPhone:
- I do not have "Larger Text" enabled in the Accessibility settings (Settings > Accessibility > Display & Text Size > Larger Text)
- I do not have any per-app settings defined for the BlueSky app
- I do not have any custom fonts installed
- I don't see any other obvious text size settings in the iOS settings.

So, AFAIK, I have not made any font size changes. If there's a specific setting you need me to check, LMK.

**Comment by ansh** [comment]
Makes sense. Seems like the reason it was happening is becuase your phone is the iPhone mini which is much smaller than the other phones. 

I've made a fix for this, tested it out as well. Here is a screenshot below. It will be released as soon as possible!

Thanks for the report :) 


<img width="487" alt="CleanShot 2023-06-06 at 15 04 39@2x" src="https://github.com/bluesky-social/social-app/assets/8207733/d42de305-7ed4-4c0c-8636-91dff43151a3">
